sdk/core/amqp-common/src/requestResponseLink.ts

```diff
@@ -98,7 +98,7 @@ export class RequestResponseLink implements ReqResLink {
     if (!options) options = {};
 
     if (!options.timeoutInSeconds) {
```

Comment:
Are we sure that when the `RequestResponseLink` for cbs is created, a custom `timeoutInSeconds` is not passed in?

Reply:
In amqp-common, for the negotiation claim, sendRequest() is called without providing any options, which is why the timeout defaults to 10s inside the sendRequest() method.

![image](https://user-images.githubusercontent.com/10452642/76792669-c06c0180-6780-11ea-81c6-36ca84e9e296.png)